import java.util.Scanner;
public class Demojava
{
	public static void main(String[] args)
	{
		Scanner sc=new Scanner(System.in);
		int a=sc.nextInt();
		int b=sc.nextInt();
		int c=sc.nextInt();
		System.out.println("Int Operations-"+(a+b*c)+","+(a*b+c)+","+(c+a/b)+","+(a%b+c));
	}
}